src/utils/schema/cleaners.ts

```diff
@@ -9,6 +10,13 @@ import type { UiSchema } from '@rjsf/core';
  * @param dirtyUiSchema
  */
 function cleanUiSchema(schema: VernaJSONSchemaType, dirtyUiSchema: UiSchema) {
+  Object.entries(schema.properties || {}).forEach(([key]) => {
+    dirtyUiSchema[key] = {
+      ...dirtyUiSchema[key],
+      'ui:ObjectFieldTemplate': Section,
+    };
+  });
```

Comment:
Add a comment to explain what you are doing please.

Furthermore, I think `Object.keys` is more appropriate than `Object.entries` in this case.

Reply:
indeed, bits of debug there